bin/oref0-radio-tune.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+source $(dirname $0)/oref0-bash-common-functions.sh || (echo "ERROR: Failed to run oref0-bash-common-functions.sh. Is oref0 correctly installed?"; exit 1)
+
+usage "$@" <<EOF
+Usage: $self
+Determine the optimum frequency to communicate with the pump and write
+the output to monitor/mmtune.json and monitor/medtronic_frequency.ini
+EOF
+
+export MEDTRONIC_PUMP_ID=`grep serial pump.ini | tr -cd 0-9`
```

Comment:
i think you should also set MEDTRONIC_FREQUENCY here
at least for WW pump from me it used 915 for me from the command line.